core/src/main/java/com/linecorp/armeria/client/DefaultClientRequestContext.java

```diff
@@ -64,6 +65,9 @@
     private long responseTimeoutMillis;
     private long maxResponseLength;
 
+    private final ContentPreviewerFactory requestContentPreviewerFactory;
+    private final ContentPreviewerFactory responseContentPreviewerFactory;
```

Comment:
How about removing these properties and using `ctx.options().request/responseContentPreviewerFactory()`, given that these properties are not likely to be accessed by users?